### PR TITLE
Prepare release 2.3.0

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- release 2.3.0
- bump Rasa SDK to 2.3

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -90,7 +90,7 @@ colorhash = "~1.0.2"
 jsonschema = "~3.2"
 packaging = ">=20.0,<21.0"
 pytz = ">=2019.1,<2021.0"
-rasa-sdk = "^2.2.0"
+rasa-sdk = "^2.3.0"
```
Comment (mprazz) [inline pyproject.toml:93]: 2.3.1 ?

Comment (wochinge) [reply pyproject.toml:93]: Mhm, pip should find out that `uvloop` isn't available on 2.3.0 and automatically install 2.3.1 on Windows systems, no?

Comment (mprazz) [reply pyproject.toml:93]: One can hope so 😅  alright, I'll trust you on this, you know more about deps than me lol

**Review by mprazz — APPROVED** [review]
🔥

File: pyproject.toml (hunk 1 above)
Comment (wochinge) [reply pyproject.toml:93]: I'll bump it. Better be safe than sorry.

**Pull request merged**